src/vbk/test/integration/test_setup.cpp

```diff
@@ -136,8 +15,8 @@ IntegrationTestFixture::IntegrationTestFixture() : TestChain100Setup()
         return util_service_impl.compareForks(left, right);
     });
     When(Method(util_service_mock, CheckPopInputs)).AlwaysReturn(true);
-    When(Method(util_service_mock, EvalScript)).AlwaysDo([&](const CScript& script, std::vector<std::vector<unsigned char>>& stack, ScriptError* serror, VeriBlock::Publications* pub, bool with_checks) -> bool {
-        return ::TestEvalScript(script, stack, serror, pub, with_checks);
+    When(Method(util_service_mock, EvalScript)).AlwaysDo([&](const CScript& script, std::vector<std::vector<unsigned char>>& stack, ScriptError* serror, VeriBlock::Publications* pub, VeriBlock::Context* ctx, VeriBlock::PopTxType* type, bool with_checks) -> bool {
+        return VeriBlock::EvalScriptImpl(script, stack, serror, pub, ctx, type, with_checks);
```

Comment:
We should use TestEvalScript instead of the origin implementation of the EvalScriptImpl, because in these test cases in the integration tests we have only one ATV without any VTB in the pop transaction, and EvalScriptImpl produces error when there are no VTBs